### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by virtue3** [ericblade/quagga2#526] [General]
Just wanted to say thank you.  This project dramatically saved my butt.  I am not able to financially thank you currently but I deeply appreciate the work you have done.  This project is significantly more reliable and at least twice as fast as the original quagga.  

Again, thanks Eric!

**Comment by lampsbr** [comment]
Hell yeah, I agree

**Comment by ericblade** [comment]
Wow, I've not done comparisons against the original, as my hardware is much much faster than it used to be back when I forked :-D

But very glad to hear other people are getting good use from it :)
